packages/core/admin/ee/admin/pages/Roles/CreatePage/index.js

```diff
@@ -1,36 +1,52 @@
-import React, { useState, useRef } from 'react';
-import { Header } from '@buffetjs/custom';
-import { Padded } from '@buffetjs/core';
-import moment from 'moment';
-import { Formik } from 'formik';
-import { get, isEmpty } from 'lodash';
-import { useIntl } from 'react-intl';
-import { HeaderLayout, Button } from '@strapi/parts';
-import { AddIcon, EditIcon } from '@strapi/icons';
 import {
-  BaselineAlignment,
   CheckPagePermissions,
   request,
-  useTracking,
   useNotification,
   useOverlayBlocker,
+  useTracking,
 } from '@strapi/helper-plugin';
+import {
+  Box,
+  Button,
+  ContentLayout,
+  Grid,
+  GridItem,
+  HeaderLayout,
+  Row,
+  Stack,
+  Text,
+  Textarea,
+  TextInput,
+  Main,
+} from '@strapi/parts';
+import { Formik } from 'formik';
+import { get, isEmpty } from 'lodash';
+import moment from 'moment';
+import React, { useRef, useState } from 'react';
+import { useIntl } from 'react-intl';
 import { useHistory, useRouteMatch } from 'react-router-dom';
-import adminPermissions from '../../../../../admin/src/permissions';
-import { useFetchPermissionsLayout, useFetchRole } from '../../../../../admin/src/hooks';
-import PageTitle from '../../../../../admin/src/components/SettingsPageTitle';
-import FormCard from '../../../../../admin/src/components/FormBloc';
-import { ButtonWithNumber } from '../../../../../admin/src/components/Roles';
-import SizedInput from '../../../../../admin/src/components/SizedInput';
+import styled from 'styled-components';
 import Permissions from '../../../../../admin/src/components/Roles/Permissions';
-
+import PageTitle from '../../../../../admin/src/components/SettingsPageTitle';
+import { useFetchPermissionsLayout, useFetchRole } from '../../../../../admin/src/hooks';
+import adminPermissions from '../../../../../admin/src/permissions';
 import schema from './utils/schema';
 
+const UsersRoleNumber = styled.div`
+  border: 1px solid ${({ theme }) => theme.colors.primary200};
+  background: ${({ theme }) => theme.colors.primary100};
+  padding: ${({ theme }) => `${theme.spaces[2]} ${theme.spaces[4]}`};
+  color: ${({ theme }) => theme.colors.primary600};
+  border-radius: ${({ theme }) => theme.borderRadius};
+  font-size: ${12 / 16}rem;
+  font-width: bold;
+`;
+
 const CreatePage = () => {
   const toggleNotification = useNotification();
   const { lockApp, unlockApp } = useOverlayBlocker();
   const { formatMessage } = useIntl();
-  const [isSubmiting, setIsSubmiting] = useState(false);
+  const [, setIsSubmiting] = useState(false);
```

Comment:
Why are you not using the state anymore?

It was used to show a loader in the header button when submitting the form.

Reply:
I don't seem to find any loader component in the headerLayout now. is it missing ?

Reply:
nvm I found the component :)